src/enums/issue-type.enum.ts

```diff
@@ -0,0 +1,7 @@
+// @todo fix this "false" report from ESLint
+// Or I just missed something IDK
+// eslint-disable-next-line no-shadow
+export enum IssueTypeEnum {
```

Comment:
nit: I'm not a fan of the all caps style. can we make this match the typescript handbook style? https://www.typescriptlang.org/docs/handbook/enums.html